packages/tools/src/synchronizers/callbacks/cameraSyncCallback.ts

```diff
@@ -34,9 +37,15 @@ export default function cameraSyncCallback(
   const tViewport = renderingEngine.getViewport(targetViewport.viewportId);
 
   // TODO: only sync in-plane movements if one viewport is a stack viewport
+  if (camera.parallelProjection) {
+    const sViewport = renderingEngine.getViewport(sourceViewport.viewportId);
+    const srcDelta = sViewport.getPan();
 
-  // Todo: we shouldn't set camera, we should set the focalPoint
-  // to the nearest slice center world position
-  tViewport.setCamera(camera);
+    // Do the zoom first, as the pan is relative to the zoom level
+    tViewport.setZoom(sViewport.getZoom());
+    tViewport.setPan(srcDelta);
```

Comment:
What happens to the scroll sync? Isn't scrolling sync handled by the cameraSyncCallback? Previously it was handled by .setCamera but now it is not (since only we setZoom and setPan here)

Reply:
We probably need a separate call for that - options are:
1. Can pass scroll sync as the 3rd parameter in pan (eg setPan(Point3) instead of Point2)
2. Have separate setScroll/getScroll methods

It looks like almost all of this is defined in the scroll.ts file, plus scrollTool, but if this change is part of the overall change, then the scrolling should be defined as core functionality in Viewport, so that there aren't multiple methods dealing with scrolling.  Do you want this included right now?